bench/bench.c

```diff
@@ -391,7 +393,7 @@ int main(int argc, char *argv[]) {
   strncpy(usage, "Usage: bench [blosclz | lz4 | lz4hc | snappy | zlib | zstd] "
           "[noshuffle | shuffle | bitshuffle] "
           "[single | suite | hardsuite | extremesuite | debugsuite] "
-          "[nthreads] [bufsize(bytes)] [typesize] [sbits]", 255);
+          "[nthreads] [bufsize(bytes)] [typesize] [sbits] [fast | safe]", 255);
```

Comment:
For consistency, rename this to '[safe | unsafe]'

Reply:
Done.